src/overmap.h

```diff
@@ -46,6 +46,7 @@ class building_bin {
         building_bin() {};
         void add( const overmap_special_id &building, int weight );
         overmap_special_id pick() const;
+        std::vector<std::string> all;
```

Comment:
Note: It would be more clear to use `std::vector<oter_id>` here, but this causes error during game loading, so I stuck with `std::string`.